jwt.js

```diff
@@ -128,13 +130,38 @@ module.exports = ({
                 request => request.headers.authorization && request.headers.authorization.match(/^basic\s+(.+)$/i)?.[1]
             ));
 
+            server.auth.scheme('swagger.apiKey.custom', (cacheConfig) => ({
+                authenticate: async(req, h) => {
+                    const dest = req.route.settings?.app?.securityRequestMethod;
+                    try {
+                        if (!dest) {
+                            throw new Error('Missing method in <uri>.method.x-options.app.securityRequestMethod');
+                        }
+                        const [{actorId}] = await brokerRequest({
+                            headers: req.headers,
+                            params: req.params,
+                            payload: req.payload,
+                            query: req.query,
+                            path: req.path,
+                            channel: 'web'
+                        }, {method: dest});
+
+                        return h.authenticated({credentials: {actorId}});
```

Comment:
maybe check if there is actorId and throw if not. The developer implementing the method should generally guarantee that but just in case....